### PR TITLE
Reordering of AMD readings, Java benchmark with Panama, README changes

### PR DESCRIPTION
**Pull request by Jakobzs**

### CODE REVIEW

**Review by sabj17 — COMMENTED** [review]
More comments plz

File: rapl-bin/src/main.rs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 use anyhow::Result;
-use rapl_lib::rapl;
+use rapl_lib::ffi::{start_rapl, stop_rapl};
 use std::{thread, time::Duration};
 
 pub fn bench_test(n: i32) -> i32 {
```
Comment (Mads518d) [inline rapl-bin/src/main.rs:5]: hvad er "i32"?

**Review by Mads518d — APPROVED** [review]
Jeg ved ikke nok om RAPL til at kunne sige om det vi gør er rigtigt eller forkert så jeg stoler på jer

File: benchmarks/fibjava/Bench.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        */
+
+        try {
+            start_rapl.invoke();
+        } catch (Throwable e) {
+            e.printStackTrace();
+        }
+
+        int result = fib(n);
+        System.out.println(result);
```
Comment (Villiamfj) [inline benchmarks/fibjava/Bench.java]: De andre benches printer ikke deres resultat.

File: benchmarks/fibjava/Bench.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        // works without arena as seen below, but not sure if it is correct to do so
+        // the code is commented out here in case it is needed later
+
+        try (Arena arena = Arena.ofConfined()) {
+            start_rapl.invoke();
+        } catch (Throwable e) {
+            e.printStackTrace();
+        }
+        */
+
```
Comment (Villiamfj) [inline benchmarks/fibjava/Bench.java]: where da loop at?

**Review by Villiamfj — CHANGES_REQUESTED** [review]

File: benchmarks/fibjava/Bench.java (hunk 2 above)
Comment (Jakobzs) [reply benchmarks/fibjava/Bench.java]: wow

File: benchmarks/fibjava/Bench.java (hunk 3 above)
Comment (Jakobzs) [reply benchmarks/fibjava/Bench.java]: wow

File: benchmarks/FibSequence/fibjava/Bench.java
Comment (sabj17) [inline benchmarks/FibSequence/fibjava/Bench.java:1]: We should consider writing some documentation for this - comments etc. (at least as a header for the file)

**Review by Villiamfj — APPROVED** [review]

**Review by sabj17 — COMMENTED** [review]
We should consider better documentation for our code. At least a short explanation in the beginning of each (important) file.

File: benchmarks/FibSequence/fibjava/Bench.java
Comment (Villiamfj) [reply benchmarks/FibSequence/fibjava/Bench.java:1]: Added some comments to the file

**Pull request merged**